### ISSUE TITLE
Filtering stops working on second view

### ISSUE THREAD

**Issue by lstoeckert** [davidegironi/advanceddatagridview#52]
I'm trying to use the Advanced DataGridView in several different forms (viewing different data) within the same project.  When the form is first viewed, the sorting and filtering works great.  If I close that form and then relaunch the same form, the grid is there but the sorting and filtering no longer work (the down arrows don't respond).  Any idea what I'm doing wrong?

**Comment by davidegironi** [comment]
Hello @lstoeckert , I've no idea, never encounter this behavior before. If you want you can take a look at dentned, there I use Advanced DataGridView in many forms without problems. https://github.com/davidegironi/dentned. Hope this helps.

**Comment by lstoeckert** [comment]
Hi Davide,

 

I’m not sure if I identified the problem or just a way around it.  In my program, I was doing a ‘showdialog’ for each form, each time I needed the form.  I added a ‘dispose’ for the form after showing it and the next time I showed it again, the ADGV worked fine.  So there must have been something left over from the previous showing that was preventing the ADGV to function correctly.

 

Thanks for your help.

 

 

Regards,

 

Lee
